solarforecastarbiter/reports/templates/html/macros.j2

```diff
@@ -108,46 +108,70 @@ forecasts/single/{{ forecast.forecast_id }}
 {% endmacro %}
 
 
-{% macro validation_table(proc_fxobs_list, quality_filters) %}
+{% macro validation_metadata_table(quality_filters) %}
 <div class="report-table-wrapper">
-<table class="table table-striped validation-table" style="width:100%;">
+<table class="table table-striped validation-table" style="width:100%;" id="data-validation-filters">
+  <caption style="caption-side:top; text-align: left">
+    Table of data validation filters
+  </caption>
+  <thead>
+    <tr class="header">
+      <th style="text-align: left;">Quality flags</th>
+      <th style="text-align: left;">Discard before resample</th>
+      <th style="text-align: left;">Resample threshold (%)</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for qfilter in quality_filters %}
+      <tr>
+        <td style="test-align: left">{{ qfilter.quality_flags | join(', ') }}</td>
+        <td style="test-align: left">{{ qfilter.discard_before_resample }}</td>
+        <td style="test-align: left">{{ qfilter.resample_threshold_percentage }}</td>
+      </tr>
+    {% endfor %}
+  </tbody>
+</table>
+</div>
+{% endmacro %}
+
+
+{% macro validation_results_table(proc_fxobs_list) %}
+<div class="report-table-wrapper">
+<table class="table table-striped validation-table" style="width:100%;" id="data-validation-results">
   <caption style="caption-side:top; text-align: left">
     Table of data validation results
   </caption>
   <thead>
     <tr class="header">
       <th style="text-align: left;">Aligned Pair</th>
       {% for proc_fxobs in proc_fxobs_list %}
-      <th style="text-align: center;">
+      <th style="text-align: center; vertical-align: middle">
         {{ proc_fxobs.name }}
       </th>
       {% endfor %}
     </tr>
     <tr class="header">
       <th style="text-align: left;">Observation</th>
       {% for proc_fxobs in proc_fxobs_list %}
-      <th style="text-align: center;">
+      <th style="text-align: center; vertical-align: middle">
         {{ proc_fxobs.original.data_object.name }}
       </th>
       {% endfor %}
     </tr>
   </thead>
   <tbody>
-    {% for qfilter in quality_filters %}
+    {% for res0 in proc_fxobs_list[0].validation_results %}
```

Comment:
This is the only part I'm worried about. Normally it seems like all the flags should be present for each proc_fxobs, but is it guaranteed? alternative is to make a simple filter to get all the unique flags from a list of processedforecastobservations.validation_results